index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Apple</title>
+    <link rel="stylesheet" href="styling/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/d4de0f4540.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <nav>
+      <ul>
+        <li><a href="#"><i class="fab fa-apple"></i></a></li>
+        <li><a href="#">Store</a></li>
+        <li><a href="#">Mac</a></li>
+        <li><a href="#">iPod</a></li>
+        <li><a href="#">iPhone</a></li>
+        <li><a href="#">iPad</a></li>
+        <li><a href="#">iTunes</a></li>
+        <li><a href="#">Support</a></li>
+        <li>
+          <div class="input-container">
+            <input type="text" name="name" id="input">
+            <i class="fas fa-search"></i>
+          </div>
+        </li>
+      </ul>
+    </nav>
+    <header id="header">
+      <div class="header-title">
+        <h1><a href="#"><img src="assets/pics/your_verse_title_small.png" alt="your_verse_title_small"></a></h1>
+        <h2><a href="#"><img src="assets/pics/your_verse_subhead_small.png" alt="your_verse_subhead_small"></a></h2>
+        <div class="inline-header">
+          <a href="#" class="h3"><img src="assets/pics/cta_your_verse_27pt.png" alt=""></a>
+        </div>
+      </div>
+    </header>
+    <aside>
+      <ul>
+        <li>
+          <a href="#"><img class="phone-pics" src="./assets/pics/promo_iphone5s.jpg" alt="promo_iphone5s"></a>
+        </li>
+        <li>
+          <a href="#"><img class="phone-pics" src="./assets/pics/promo_iphone5c.jpg" alt="promo_iphone5c"></a>
+        </li>
+        <li>
+          <a href="#"><img class="phone-pics" src="./assets/pics/promo_30_years.jpg" alt="promo_30_years"></a>
+        </li>
+        <li>
+          <a href="#"><img class="phone-pics" src="./assets/pics/promo_supplier_responsibility.jpg" alt="promo_supplier_responsibility"></a>
+        </li>
+      </ul>
+    </aside>
```

Comment:
Surround these lines in the **\<main\>** tag

Reply:
You are right I didn't think about it I just tried to avoid putting header inside main, thank you.